data-common.json

```diff
@@ -25,11 +25,13 @@
   },
   "firefox": {
     "developer": {
+      "versions": ["firefox52", "firefox53"],
```

Comment:
Is this meant to be a general purpose version mechanism for everything, so that we can consolidate all the nodeX's, for example?